extensions/integration-tests/package.json

```diff
@@ -51,12 +51,13 @@
   "devDependencies": {
     "@types/chai": "3.4.34",
     "@types/node": "^10.14.8",
+    "@types/vscode": "^1.38.0",
```

Comment:
what do you need this for?

Reply:
I was hoping to pull in types information for the following: vscode/lib/testrunner. Is there a different @types module for that?

Reply:
vscode-test

Reply:
Testing with vscode-test.

Also if we have pre-existing dependency on 'vscode' in this project then shouldn't we add dev dependency for @types/vscode as well? Alan, is it a case that the vscode itself was a redundant dependency that can be cleaned out.

Reply:
the types are included in the vscode module

Reply:
You should sync with @Charles-Gagnon 7211

Reply:
ah - good to know about the types being included by 'vscode'

Reply:
synced with @Charles-Gagnon #7211, he will go in after me and merge any changes that need merging. I am changing to use his environment variable names to be conformant.